rustls/src/builder.rs

```diff
@@ -70,7 +70,7 @@ use core::marker::PhantomData;
 ///
 /// For a client, _certificate verification_ must be configured either by calling one of:
 ///  - [`ConfigBuilder::with_root_certificates`] or
-///  - [`ConfigBuilder::dangerous.with_custom_certificate_verifier`]
+///  - [`ConfigBuilder::dangerous()`]`.with_custom_certificate_verifier`
```

Comment:
Presumably we can't link directly to `with_custom_certificate_verifier` because of the issue you mentioned in Discord w.r.t the `DangerousClientConfigBuilder` type not being visible? Do you want to fix that in this branch and then update the link at the same time?